### ISSUE TITLE
Align/anchor icons to the bottom

### ISSUE THREAD

**Issue by nick-s-b** [linuxmint/nemo#1991]
```
 * Nemo version (nemo --version)
4.0.1
 * Is issue with desktop or windowed nemo?
windowed
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
Arch
 * Graphics hardware *and* driver used
Nvidia
 * 32 or 64 bit
64bit
 ```

**Issue**
This is a really minor issue but given how `nemo` 4 has improved so much in the `Icon View` (my favorite view), I hope this small issue will be addressed at some point.

After `nemo` 4, all preview icons seem to have a fixed amount of space on the gird (a really nice change since it makes it all more consistent) but I've noticed that icons are center-aligned and preview/thumbnail images are bottom aligned. If you have a folder that has a mix of them (some files like bash scripts don't have a thumbnail for example), it looks ugly. It would be great if they were all anchored/aligned the same way. Personally, I think bottom aligned is better since the distance from the filename text to image is always uniform but I'd be happy with either option as long as it's consistent.

tl;dr: suggestion is to align icons to the bottom.

![screenshot from 2018-11-20 19-18-43-fs8](https://user-images.githubusercontent.com/7957590/48811022-dd620300-ecf9-11e8-8ba8-7bad54f8f812.png)

`nemo` is still the best FM and nothing else even comes close. Thank you for making it better and for caring!

**Comment by NikoKrause** [comment]
I think that it is bottom aligned. Looking at the `image-x-generic` icon (the right one):
![screenshot-area-2018-11-21-191038](https://user-images.githubusercontent.com/8415124/48860426-29c44600-edc1-11e8-8723-bf4cc2147e6a.png)
The empty space below is in the image itself.

See the following example:
![screenshot-area-2018-11-21-191455](https://user-images.githubusercontent.com/8415124/48860658-cd155b00-edc1-11e8-85bf-109ec70bf573.png)

**Comment by nick-s-b** [comment]
@NikoKrause You're right. Just looked at Adwaita's icons myself and it seems that they're all centered with lots of empty space around them. The biggest annoyance are folder icons which just look weird when most of the images are bottom centered. I guess I'll try to find a better icon set or make my own. Thanks again Niko!!!

**Comment by mtwebster** [comment]
Yeah I noticed this right off the bat, and am not happy about how it looks with mixed folders and thumbnails, but the only alternative was to center thumbnails vertically, and that didn't look too good either (this is how nautilus has it in their 'new' icon views) since folder icons aren't necessarily perfectly centered.  Plus it sort of puts too much detachment between the image and the text that belongs to it.

I'm happy to say this is something I've more or less learned to ignore :)
